[Chore] Update README (#39)
nimblehq/android-common-ktx: README.md

<p align="center">
  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
</p>

---

# Android Common Extensions

This repository provides a collection of common Android extensions.

[![](https://jitpack.io/v/nimblehq/android-common-ktx.svg)](https://jitpack.io/#nimblehq/android-common-ktx)

## Usage

> 1. Add JitPack to your project build.gradle

Reviewer: @hoangnguyen92dn Suggestion:

```
Add JitPack to your project `build.gradle` or `settings.gradle`
```

Author: @doannimble It should be `or` or `and`? 🤔

    - Groovy

    ```groovy
    allprojects {
      repositories {
        ...
        maven { url 'https://jitpack.io' }
       }
    }
    ```

    - Kotlin DSL

    ```kotlin
    repositories {
    	...
    	maven(url = "https://jitpack.io")
    }
    ```

> 2. Add the dependency in the application build.gradle

Reviewer: @hoangnguyen92dn Should we use `build.gradle` ?

    - Groovy

    ```groovy
    dependencies {
    	implementation 'com.github.nimblehq:android-common-ktx:<version>'
    }
    ```

    - Kotlin DSL

    ```kotlin
    implementation("com.github.nimblehq:android-common-ktx:<version>")
    ```

## Contributing

> For major changes, please open an issue first to discuss what you would like to change. Visit our [Wiki](https://github.com/nimblehq/android-common-ktx/wiki#how-to-contribute) for guidelines to contribute your ideas to our project.

Reviewer 2: ```suggestion
The contribution is welcome by opening an issue for discussion. You can visit our [Wiki](https://github.com/nimblehq/android-common-ktx/wiki#how-to-contribute) for guidelines to contribute your ideas to our project.
```
What do you think? @hoangnguyen92dn

Author: @lydiasama Nice 👍
